### ISSUE TITLE
Pop!_Shop doesn't load next page of packages if its window is larger than the list

### ISSUE THREAD

**Issue by msmoljan** [pop-os/pop#1700]
**Distribution (run `cat /etc/os-release`):**
 Pop!_OS 20.10
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
```
pop-shop:
  Installed: 3.4.2pop0~1617654967~20.10~4589784
  Candidate: 3.4.2pop0~1617654967~20.10~4589784
  Version table:
 *** 3.4.2pop0~1617654967~20.10~4589784 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu groovy/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Issue/Bug Description:**
While Pop!_Shop is maximized, opening a section on a large display/resolution (in my case 4k, 100% scaling) only loads 21 items and leaves some empty space after that, and doesn't load the rest of the items. If I resize the window to the size smaller then the list and then scroll to the end, it starts loading the other items

**Steps to reproduce (if you know):**
1. Make sure you're on a 4k display with scaling set to 100%, and run Pop!_Shop maximized (or as the only app in tiling mode)
2. Open any category, e.g. Games
3. Notice that there are only 21 items in the list, and there is some empty space below the last one (i.e. the rest of the list is not getting loaded).

**Expected behavior:**
If the window is larger than the list, items after the 21st should start loading immediately (next page in the infinite scroll?).

**Comment by leviport** [comment]
Huh, I'm seeing the same thing. If I make the window smaller and scroll down to induce more items to load, it begins loading the list items normally.

Nice find!

**Comment by msmoljan** [comment]
Yeah, it was a bit weird that there suddenly seemed to be so few packages :)

Glad to help out!
